Build fails on Arch Linux

I can't get the build working, see the following
```
$ cat .qmake.stash 
QMAKE_CXX.QT_COMPILER_STDCXX = 201402L
QMAKE_CXX.QMAKE_GCC_MAJOR_VERSION = 8
QMAKE_CXX.QMAKE_GCC_MINOR_VERSION = 2
QMAKE_CXX.QMAKE_GCC_PATCH_VERSION = 1
QMAKE_CXX.COMPILER_MACROS = \
    QT_COMPILER_STDCXX \
    QMAKE_GCC_MAJOR_VERSION \
    QMAKE_GCC_MINOR_VERSION \
    QMAKE_GCC_PATCH_VERSION
QMAKE_CXX.INCDIRS = \
    /usr/include/c++/8.2.1 \
    /usr/include/c++/8.2.1/x86_64-pc-linux-gnu \
    /usr/include/c++/8.2.1/backward \
    /usr/lib/gcc/x86_64-pc-linux-gnu/8.2.1/include \
    /usr/local/include \
    /usr/lib/gcc/x86_64-pc-linux-gnu/8.2.1/include-fixed \
    /usr/include
QMAKE_CXX.LIBDIRS = \
    /usr/lib/gcc/x86_64-pc-linux-gnu/8.2.1 \
    /usr/lib \
    /lib
```
`make` fails with the following:
```
[...] # snipped
g++ -Wl,-O1 -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -o ../svn-all-fast-export ruleparser.o repository.o svn.o main.o CommandLineParser.o   -L -lsvn_fs-1 -lsvn_repos-1 -lapr-1 -lsvn_subr-1 -lQt5Core -lpthread   
/usr/bin/ld: svn.o: undefined reference to symbol 'svn_fs_file_contents'
/usr/bin/ld: /usr/lib/libsvn_fs-1.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:203: ../svn-all-fast-export] Error 1
make[1]: Leaving directory '/home/kartik/Projects/svn2git/src'
make: *** [Makefile:47: sub-src-make_first] Error 2
```

--- a/src/src.pro
+++ b/src/src.pro
@@ -28,7 +28,7 @@ DEPENDPATH += .
 QT = core
 
 INCLUDEPATH += . $$SVN_INCLUDE $$APR_INCLUDE
-!isEmpty($$SVN_LIBDIR): LIBS += -L$$SVN_LIBDIR
+!isEmpty(SVN_LIBDIR): LIBS += -L$$SVN_LIBDIR
 LIBS += -lsvn_fs-1 -lsvn_repos-1 -lapr-1 -lsvn_subr-1
 
 # Input

